content/api/v18/async_hooks.en.md

```diff
@@ -374,7 +373,7 @@ createHook({
     const eid = executionAsyncId();
     fs.writeSync(
       stdout.fd,
-      `$type($asyncId): trigger: $triggerAsyncId execution: $eid\n`);
+      `${type}(${asyncId}): trigger: ${triggerAsyncId} execution: ${eid}\n`);
```

Comment:
Did changes happen on upstream again?

Reply:
https://raw.githubusercontent.com/nodejs/node/main/doc/api/async_hooks.md

Reply:
Actually, this change is something that the current changes in the script fix. Previous iteration had this bug of making changes within code-blocks

Reply:
I think there might be a bug with the current script, and this is now fixing it. The other sync change didn't bring this change in from upstream

Reply:
Yup, I believe this current PR fixes that bug.